tests/tests_api/test_api.py

```diff
@@ -102,3 +68,77 @@ def test_throw_error_at_source_registration_if_error(
         kirby_app.add_destination(MagicMock(id=1))
     with pytest.raises(ClientError):
         kirby_app.add_destination(MagicMock(id=-1))
+
+
+@pytest.mark.parametrize(
+    "error_code, error_value, error_type",
+    [(500, "Server Error", ServerError), (400, "Client Error", ClientError)],
+)
+@patch("requests.session")
+def test_throw_error_at_source_registration_if_error(
+    session_mock,
+    kirby_expected_env,
+    kirby_hidden_env,
+    error_code,
+    error_value,
+    error_type,
+):
+    session_mock.return_value.get.return_value = MagicMock(
+        status_code=error_code, json=MagicMock(return_value={error_value})
+    )
+    session_mock.return_value.patch.return_value = MagicMock(
+        status_code=200, json=MagicMock(return_value={})
+    )
+    kirby_app = Kirby(kirby_expected_env)
+    with pytest.raises(error_type):
+        kirby_app.add_source(MagicMock(id=1))
+
+
+def test_it_raise_error_if_bad_usage_of_testing_mode():
+    kirby_app = Kirby({}, testing=True)
+    with pytest.raises(NotImplementedError):
+        kirby_app.get_topic_id(topic_name="")
+
+
+@pytest.mark.integration
+@pytest.mark.skipif(
+    not os.getenv("KIRBY_WEB_SERVER"),
+    reason="missing KIRBY_WEB_SERVER environment",
+)
+def test_it_get_topic_id(kirby_app, db_topics):
```

Comment:
Did you just add `_it_` to silly short names just so it's there? 😄 

If that test was breaking, would that name give me enough information to know what is broken?

Reply:
The tests had been renamed, I know how to give a better name to a test now